lib/src/Uncertainty/Algorithm/Transformation/MarginalTransformation/MarginalTransformationEvaluation.cxx

```diff
@@ -38,19 +38,20 @@ MarginalTransformationEvaluation::MarginalTransformationEvaluation():
   EvaluationImplementation(),
   inputDistributionCollection_(),
   outputDistributionCollection_(),
-  direction_()
+  direction_(FROMTO)
 {
   // Nothing to do
 }
 
 /* Parameter constructor */
 MarginalTransformationEvaluation::MarginalTransformationEvaluation(const DistributionCollection & inputDistributionCollection,
     const DistributionCollection & outputDistributionCollection,
+    const UnsignedInteger direction,
```

Comment:
@jschueller If the first argument is called inputDistributionCollection and the second outputDistributionCollection, I don't see the interest of specifying the direction. It should be FROMTO or the arguments should be renamed. What is the use case justifying to swap the role of the arguments through the constructor using a flag instead of swaping the arguments?